content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -268,19 +268,46 @@ Datadog.setUserInfo(id: "1234", name: "John Doe", email: "john@doe.com")
 
 You can use the following properties in `Datadog.Configuration` when creating the Datadog configuration to initialize the library:
 
+`clientToken`
+: Either the RUM client token (which supports RUM, Logging, and APM) or regular client token, (which supports Logging and APM).
+
+`env`
+: The environment name that is sent to Datadog. This can be used to filter events on different environments (such as `staging` or `production`).
```

Comment:
```suggestion
: The environment name that is sent to Datadog. This can be used to filter events by different environments (such as `staging` or `production`).
```